SwiftWisdom/Core/Version/Version.swift

```diff
@@ -33,45 +33,47 @@ extension Version : CustomStringConvertible {
     }
 }
 
-extension Version : Equatable {}
+// MARK: Operator Overloads
 
-public func ==(lhs: Version, rhs: Version) -> Bool {
-    guard lhs.major == rhs.major else { return false }
-    guard lhs.minor == rhs.minor else { return false }
-    guard lhs.patch == rhs.patch else { return false }
-    return true
-}
+extension Version : Equatable {
+    public static func ==(lhs: Version, rhs: Version) -> Bool {
+        guard lhs.major == rhs.major else { return false }
+        guard lhs.minor == rhs.minor else { return false }
+        guard lhs.patch == rhs.patch else { return false }
+        return true
+    }
 
-public func !=(lhs: Version, rhs: Version) -> Bool {
-    return !(lhs == rhs)
+    public static func !=(lhs: Version, rhs: Version) -> Bool {
+        return !(lhs == rhs)
```

Comment:
I'd opt for "return lhs != rhs" sine I think it's a little more clear, but to each their own.